workflow-ui/compose/src/androidTest/java/com/squareup/workflow1/ui/compose/ComposeLifecycleOwnerTest.kt

```diff
@@ -32,7 +31,9 @@ class ComposeLifecycleOwnerTest {
     composeTestRule.setContent {
       parentLifecycle.currentState = RESUMED
       childLifecycleOwner = rememberChildLifecycleOwner(parentLifecycle)
-      CompositionLocalProvider(LocalLifecycleOwner provides childLifecycleOwner) {
+      CompositionLocalProvider(
+        androidx.lifecycle.compose.LocalLifecycleOwner provides childLifecycleOwner
```

Comment:
Why remove this import?

Reply:
just didn't get the import in after the change to the androidx.lifecycle version. will add.